go.mod

```diff
@@ -10,10 +10,11 @@ require (
 	github.com/hashicorp/consul v1.8.0
 	github.com/hashicorp/consul/sdk v0.5.0
 	github.com/hashicorp/go-checkpoint v0.5.0
+	github.com/hashicorp/go-cleanhttp v0.5.1
```

Comment:
This isn't probably an issue with a project this size but we eventually might want to consider breaking out module updates that aren't directly related to the changeset.

Reply:
I ran `go mod tidy` as a part of these changes, it looks like it picked up one that was missing for some time

```sh
$ go mod why github.com/hashicorp/go-cleanhttp
# github.com/hashicorp/go-cleanhttp
github.com/hashicorp/consul-terraform-sync/e2e/internal
github.com/hashicorp/go-cleanhttp
```

This would be relevant to the `stretchr/testify` version update below 😛 -- however since it is only used for testing I think I'll keep it as a part of this PR